code/game/gamemodes/cult/talisman.dm

```diff
@@ -1,7 +1,9 @@
 /obj/item/paper/talisman
-	icon_state = "paper_talisman"
-	var/imbue = null
 	info = "<center><img src='talisman.png'></center><br/><br/>"
+	var/imbue = null
+
+/obj/item/paper/talisman/update_contents_overlays()
+	add_overlay(global.cached_paper_overlays["paper_talisman"])
```

Comment:
Not sure I like this. Should ideally use a getter, or check that there's something cached before passing it to add_overlay().

Reply:
I can do that.